EXAMPLE git clone all with popup.py

```diff
@@ -0,0 +1,119 @@
+    git_login_form_response = system.message(
+        subject='Input Git repository information to connect',
+        message_type = 'POPUP',
+        body={
+            'type': 'object',
+            'properties': {
+                'username': {
+                    'element': 'string',
+                    'type': 'string',
+                    'label': 'Enter your username for the GIT repository:',
+                    'order': 1,
+                },
+                'password': {
+                    'element': 'password',
+                    'type': 'string',
+                    'label': 'Enter your password for the GIT repository:',
+                    'order': 2,
+                },
+                # Here an example input is shown in the field, but its value is empty: 
+                'repository url': {
+                    'element': 'string',
+                    'type': 'string',
+                    'label': 'Enter the URL for the repository you want to clone into Cloudomation:',
+                    'example': 'https://repository.url/user/repository.git',
+                    'order': 3,
+                },
+                # Here a default value is provided with the field: 
+                'reference': {
+                    'element': 'string',
+                    'type': 'string',
+                    'label': 'Enter the "branch" you want to clone from or a "commit sha":',
+                    'default': 'develop',
+                    'order': 4,
+                },
+                'Ok': {
+                    'element': 'submit',
+                    'label': 'OK',
+                    'type': 'boolean',
+                    'order': 5,
+                },
+            },
+            'required': [
+                'username',
+                'password',
+                'repository url',
+                'reference',
+            ],
+        },
+    ).wait().get('response')
+    git_username = git_login_form_response['username'] 
+    git_password = git_login_form_response['password']
+    git_repo_url = git_login_form_response['repository url']
+    git_reference = git_login_form_response['reference']
+
+    # Now we use a "GIT" task.                                      
+    # The git 'get' command fetches the content of the repository.
+    # Since no files_path is passed, the files will be returned in the
+    # output_value of the task
+    files = None
+    try:
+        files = this.task(
+            'GIT',
+            command='get',
+            repository_url=git_repo_url,
+            # httpCookie=repo_info.get('httpCookie'),
+            ref=git_reference,
+            username=git_username,
+            password=git_password,
+        ).get('output_value')['files']
+    except flow_api.exceptions.DependencyFailedError as err:
+        files = None
+        # send the error message to the output of this execution:
+        this.set_output(err)
```

Comment:
When the GIT task fails, end the execution here with an error
```suggestion
        # send the error message to the output of this execution:
        return this.error(repr(err))
```